internal/providers/azure/azure.go

```diff
@@ -242,7 +242,9 @@ func isCdromPresent(logger *log.Logger, devicePath string) bool {
 		logger.Info("failed to open config device: %v", err)
 		return false
```

Comment:
internal/providers: Ignore close() errors
c91852268bc43967e64dd1f5695d73b41c1591fb

Missing lint error details in commit message.